app/services/mailer/mailer.js

```diff
@@ -0,0 +1,23 @@
+const config = require('../../../config');
+const nodemailer = require('nodemailer');
+
+exports.send = (to, content) => {
+  const transporter = nodemailer.createTransport(config.common.mailer);
+  return transporter.sendMail({
+    from: `"Wolox Training 👤" <${config.common.mailer.auth.user}>`,
+    to,
+    subject: content.subject,
+    html: `<!DOCTYPE html>
```

Comment:
usually we do this in a separate file

Reply:
Ok, i moved the layout as a new template & compose it with the signup template
No more hardcoded html in mailer